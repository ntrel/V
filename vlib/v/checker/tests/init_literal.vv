i := 3
_ = []int{len: 3.3}
_ = []int{cap: &i}
_ = chan int{cap: [i]}
