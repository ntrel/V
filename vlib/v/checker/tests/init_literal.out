vlib/v/checker/tests/init_literal.vv:2:16: error: array `len` needs to be an int
    1 | i := 3
    2 | _ = []int{len: 3.3}
      |                ~~~
    3 | _ = []int{cap: &i}
    4 | _ = chan int{cap: [i]}
vlib/v/checker/tests/init_literal.vv:3:16: error: array `cap` needs to be an int
    1 | i := 3
    2 | _ = []int{len: 3.3}
    3 | _ = []int{cap: &i}
      |                ^
    4 | _ = chan int{cap: [i]}
vlib/v/checker/tests/init_literal.vv:4:19: error: channel `cap` needs to be an int
    2 | _ = []int{len: 3.3}
    3 | _ = []int{cap: &i}
    4 | _ = chan int{cap: [i]}
      |                   ~~~
